### ISSUE TITLE
Migrate deploymentPipeline to work with v2.0

### ISSUE THREAD

**Issue by bsquizz** [RedHatInsights/insights-pipeline-lib#46]
Since the image has changed the job now fails with:

```
[deployPlatform] Running shell script
+ python3.6 -m venv /home/jenkins/.venv
/home/jenkins/workspace/ops/deployPlatform@tmp/durable-47e8d8ef/script.sh: line 2: python3.6: command not found
```

**Comment by quarckster** [comment]
It should be fixed now because `iqe-base` switched to `centos:7`

**Comment by quarckster** [comment]
On the other hand, it's not clear for me why we need to create a new virtual environment inside `iqe-core` container.

**Comment by bsquizz** [comment]
Addressed with #50
